yt_dlp/extractor/la7.py

```diff
@@ -63,7 +63,7 @@ def _real_extract(self, url):
             url = '%s//%s' % (self.http_scheme(), url)
 
         webpage = self._download_webpage(url, video_id)
-        video_path = self._search_regex(r'(/content/.*?).mp4', webpage, 'video_path')
+        video_path = self._search_regex(r'akamaihd\.net/i(?:/,)?(/content/.*?)\.mp4', webpage, 'video_path')
```

Comment:
In the source page `akamaihd.net/i` is just used as a fake placeholder URI that will be replaced with the real one according to the logic reported below.

Thus I wouldn't match this URI since it has high chance to be changed in the future. Instead I would match the url that ends with m3u8.
```
self._search_regex(r'(/content/\S*?)\.mp4\S*?m3u8', webpage, 'video_path')
```


URI replace logic used today by `la7.it`:

```
         case 'mp4':
                e.akamai.mp4 = n.replace('vodpmd.la7.it.edgesuite.net/', 'awsvodpkg.iltrovatore.it/'),
                e.akamai.mp4.indexOf('http:') < 0 ? e.akamai.mp4 = 'https:' + e.akamai.mp4 : e.akamai.mp4 = e.akamai.mp4.replace('http:', ''),
                e.ilTrovatore.mp4 = n.replace('vodpmd.la7.it.edgesuite.net/', 'vodpkg.iltrovatore.it/local/mp4/');
                break;
              case 'm3u8':
                e.akamai.m3u8 = n.replace('la7-vh.akamaihd.net/i/', 'awsvodpkg.iltrovatore.it/local/hls/'),
                e.akamai.m3u8 = e.akamai.m3u8.replace('csmil', 'urlset'),
                e.akamai.m3u8 = 'https:' + e.akamai.m3u8,
                e.ilTrovatore.m3u8 = n.replace('la7-vh.akamaihd.net/i/', 'vodpkg.iltrovatore.it/local/hls/'),
                e.ilTrovatore.m3u8 = e.ilTrovatore.m3u8.replace('csmil', 'urlset'),
                e.akamai.dash = e.akamai.m3u8.replace('local/hls', 'local/dash/'),
                e.akamai.dash = e.akamai.dash.replace('master.m3u8', 'manifest.mpd'),
                e.ilTrovatore.dash = e.ilTrovatore.m3u8.replace('local/hls', 'local/dash/'),
                e.ilTrovatore.dash = e.ilTrovatore.dash.replace('master.m3u8', 'manifest.mpd');
                break;
              case 'f4m':
                break;
              case 'dash':
                e.akamai.dash = n.replace('la7-vh.akamaihd.net/i/', 'awsvodpkg.iltrovatore.it/local/dash/'),
                e.akamai.dash = e.akamai.dash.replace('csmil', 'urlset'),
                e.akamai.dash = e.akamai.dash.replace('master.m3u8', 'manifest.mpd'),
                e.akamai.dash = e.akamai.dash,
                e.ilTrovatore.dash = n.replace('la7-vh.akamaihd.net/i/', 'vodpkg.iltrovatore.it/local/dash/'),
                e.ilTrovatore.dash = e.ilTrovatore.dash.replace('csmil', 'urlset'),
                e.ilTrovatore.dash = e.ilTrovatore.dash.replace('master.m3u8', 'manifest.mpd')
            }
          }
 ```